modules/arm_plugin/src/arm_converter/arm_converter_pool.cpp

```diff
@@ -57,24 +59,35 @@ template<> Converter::Conversion::Ptr Converter::Convert(const opset::MaxPool& n
 }
 
 template<> Converter::Conversion::Ptr Converter::Convert(const ngraph::op::v8::MaxPool& node) {
-    auto make = [&] (auto refFunction) {
-    return this->MakeConversion(refFunction,
-                                node.input(0),
-                                node.output(0),
-                                node.output(1),
-                                node.get_input_shape(0),
-                                node.get_output_shape(0),
-                                node.get_kernel(),
-                                node.get_strides(),
-                                node.get_dilations(),
-                                node.get_pads_begin(),
-                                node.get_pads_end(),
-                                node.get_axis());
-    };
-    return CallSwitch(
-        AP_WRAP(make, ngraph::runtime::reference::max_pool),
-        node.input(0), allTypes,
-        node.output(1), indexTypes);
+    if ((node.get_input_shape(0).size() == 4) &&
+       (node.get_output_element_type(1) == ngraph::element::u32) &&
+       (node.get_kernel() == ngraph::Shape{2, 2}) &&
+       (node.get_dilations() == ngraph::Strides{1, 1}) &&
+       (node.get_axis() == 0)) {
```

Comment:
I think such a strong restriction is too tough here. We could use ACL whenever there are no consumers for the second output

Reply:
This change could be postponed to a separate PR as discussed